lms/djangoapps/learner_dashboard/tests/test_utils.py

```diff
@@ -0,0 +1,28 @@
+""" Unit test module covering utils module
+"""
+
+import ddt
+from django.test import TestCase
+
+from lms.djangoapps.learner_dashboard.utils import (
+    FAKE_COURSE_ID_FOR_URL,
+    url_remove_fake_course_id)
+
+
+@ddt.ddt
+class TestUrlRemoveFakeId(TestCase):
+    """ The test case class covering the util function 'url_remove_fake_course_id'
+    """
+    @ddt.data(['path1'], ['path1', 'path2'], [])
+    def test_remove_fake_id_from_url(self, path_array):
```

Comment:
This is a [change detector test](http://googletesting.blogspot.com/2015/01/testing-on-toilet-change-detector-tests.html). I'm not sure about the value it provides in its current form.